web-ui/src/pages/BirthdayReportPage.jsx

```diff
@@ -13,33 +13,47 @@ import SearchBirthdayAnniversaryResults from '../components/search-results/Searc
 import { sortBirthdays } from '../context/util';
 
 import { selectCsrfToken } from '../context/selectors';
+import { useQueryParameters } from '../helpers/query-parameters';
 
 const months = [
-  { month: 'January' },
```

Comment:
Using objects with a `month` property was making this code more complicated than it needed to be. This is now just an array of strings.